src/main/io/msp_protocol.h

```diff
@@ -164,6 +164,8 @@
 
 #define MSP_TRANSPONDER_CONFIG          82 //out message         Get transponder settings
 #define MSP_SET_TRANSPONDER_CONFIG      83 //in  message         Set transponder settings
+#define MSP_FLASHCHIP_CONFIG            84 //out  message        Get flash chip settings
```

Comment:
IDs 84 & 85 are used already by https://github.com/cleanflight/cleanflight/pull/2112/files.
I also used 86 & 87 in https://github.com/cleanflight/cleanflight/pull/2173/commits/14471c16ff3a8bc14af3fa36eddee09b0cbe3a19 (merged)
88 & 89 should be available.